projects/dotnet.yml

```diff
@@ -59,3 +59,41 @@ results:
       - plain
       - sha1
       - sha256
+
+  ferretdb-postgresql-secured:
+    stats:
+      fail: 5
+      pass: 1
+    fail:
+      - noauth # Authentication failed
+
+      # TODO https://github.com/FerretDB/FerretDB/issues/4065
+      # TODO https://github.com/FerretDB/dance/issues/889
+      # Unsupported OP_QUERY command: saslStart. The client driver may require an upgrade.
+      - plain
+
+      # TODO https://github.com/FerretDB/FerretDB/issues/4065
+      # TODO https://github.com/FerretDB/dance/issues/889
+      # Unsupported OP_QUERY command: saslContinue. The client driver may require an upgrade.
```

Comment:
Yes true, it indicates that we need to implement OpQuery for `saslStart` and `saslContinue`. Supported `OpQuery` are `hello` and `isMaster` at the moment.